src/lib/components/Alert/Alert.tsx

```diff
@@ -43,7 +39,7 @@ export interface AlertProps extends PropsWithChildren<Omit<ComponentProps<'div'>
 export const Alert: FC<AlertProps> = ({
   additionalContent,
   children,
-  color = 'info',
+  color = 'pink',
```

Comment:
Can you revert the change to this line? We're good to go with this PR at that point

Reply:
done!